src/Polysemy/Internal.hs

```diff
@@ -300,18 +300,20 @@ raiseUnder3 = hoistSem $ hoist raiseUnder3 . weakenUnder3
 
 
 ------------------------------------------------------------------------------
--- | Lift an effect into a 'Sem'. This is used primarily via
+-- | Embed an effect into a 'Sem'. This is used primarily via
 -- 'Polysemy.makeSem' to implement smart constructors.
 send :: Member e r => e (Sem r) a -> Sem r a
 send = liftSem . inj
 {-# INLINE[3] send #-}
 
 
 ------------------------------------------------------------------------------
--- | Lift a monadic action @m@ into 'Sem'.
-sendM :: Member (Lift m) r => m a -> Sem r a
-sendM = send . Lift
-{-# INLINE sendM #-}
+-- | Embed a monadic action @m@ in 'Sem'.
+--
+-- TODO(sandy): @since
```

Comment:
Not sure if this actually needs a `@since`, added just in case.

Reply:
This thing has been around since v0, but I guess it doesn't hurt to be explicit about that! Good catch!

Reply:
I was thinking more does it needs a since for whatever version this will be in, because it's renamed.

Reply:
Yup, you're right! That's what I get for reviewing just before bed.